src/MetricPusher.cs

```diff
@@ -10,6 +10,7 @@ namespace Prometheus.Client.MetricPusher
     public class MetricPusher : IMetricPusher
```

Comment:
Add `IDisposable` to implementation instead of interface

```suggestion
    public class MetricPusher : IMetricPusher, IDisposable
```